### PR TITLE
All files for SparkFun LumiDrive and Redboard Turbo 

### PR DESCRIPTION
**Pull request by edspark**
There's two commits here. The first one adds config files to the atmel-samd boards file for the SparkFun LumiDrive and Redboard Turbo. The other commit adds the hardware profile of the flash chip used on these boards to devices.h. Let me know if you want me to split these commits and/or whatever else you need for a smooth merge. 

Thanks!

### CODE REVIEW

File: ports/atmel-samd/boards/sparkfun_lumidrive/mpconfigboard.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#define SPI_FLASH_MOSI_PIN   &PIN_PA08
+#define SPI_FLASH_MISO_PIN   &PIN_PA14
+#define SPI_FLASH_SCK_PIN    &PIN_PA09
+#define SPI_FLASH_CS_PIN     &PIN_PA13
+
+// All port(pin) definitions should be deleted unless PA06 -- SERCOM0/PAD[2]
+// Will need to scour the repo to figure out why this was done. 
+// I'm guessing these ports are defined elsewehere by default rather than needing to
+// repeatedly define them. Saves time to name outliers than participants in
+// some cases.
```
Comment (tannewt) [inline ports/atmel-samd/boards/sparkfun_lumidrive/mpconfigboard.h]: These are definitely poorly named. They ensure some pins are not automatically reset. We're moving to a different way of doing it so if your board works as expected then this should be fine.

File: ports/atmel-samd/boards/sparkfun_redboard_turbo/mpconfigboard.h [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+// supported/defined flash module. 
+//
+// This board - the LUMIDrive - uses a W25Q32FV component. It's 
+// definition was added to devices.h, based on the winbond spec sheet.
+//
+// You can list N number of devices - just update the count define and comma 
+// seperate the EXTERNAL_FLASH_DEVICES list of devices.  
+                               
+// #define EXTERNAL_FLASH_DEVICE_COUNT 1
+// #define EXTERNAL_FLASH_DEVICES W25Q32FV
```
Comment (tannewt) [inline ports/atmel-samd/boards/sparkfun_redboard_turbo/mpconfigboard.h]: Need this comment still since it's moved files?

Comment (edspark) [reply ports/atmel-samd/boards/sparkfun_redboard_turbo/mpconfigboard.h]: Derp, also forgot to remove this extraneous stuff. Great catch! Resolved in latest commit.

File: ports/atmel-samd/boards/sparkfun_lumidrive/mpconfigboard.h (hunk 1 above)
Comment (edspark) [reply ports/atmel-samd/boards/sparkfun_lumidrive/mpconfigboard.h]: End of day here for me, didn't mean to resolve without comment. Sorry about the extraneous comments. I removed them in latest commit.

**Comment by edspark** [comment]
Added our boards to the travis.yml and removed the extraneous comments.

**Pull request merged**